### DISCUSSION TITLE
Bluetooth Receiver to Aux Output

### DISCUSSION THREAD

**Discussion by uber1geek** [mikebrady/shairport-sync#1899] [Q&A]
Hi All,

Thanks for maintaining this super awesome project.

I stumbled here while trying to revive my old 5.1 speaker system and connect it to my tv via bluetooth, currently I have configured AirPlay2 and it works geat but how do I enable Bluetooth Receiver so that I can output audio via Aux to my speakers

Thanks

**Comment by mikebrady** [comment]
Thanks for the question. There is quite a bit about this in the Issues, and some people have had some success, but it is tricky to get Bluetooth to work.
